Explorer/Controller/CreateLogViewController.swift

```diff
@@ -46,6 +53,15 @@ class CreateLogViewController: UIViewController {
         createLogView.saveDiveButton.addTarget(self, action: #selector(saveDive), for: .touchUpInside)
         
         installConstraints()
+        
+        createLogView.diveNumberTextField.delegate = self
```

Comment:
I'm working on this feedback but have some questions:
1. I had a done button in the DatePickerView file already but after seeing your comment, I changed it to use the toolbar. Originally, I added the button in the DatePickerView but the action (setDate) in the CreateLogViewController file. Now I assign the action in the DatePickerView file (but the function is still in CreateLogViewController). Is this the best way to handle this? I have this change ready to commit if so. Here's how I set the action in DatePickerView:
```
let doneButton = UIBarButtonItem(title: "Done", style: .done, target: self, action: #selector(CreateLogViewController.setDate))
```

2. I'm not sure how a "Next" button should flow with the depth slider and entry and water type toggles. Should I just skip over them? Should I not have a done or next button so the user doesn't skip them accidentally? I assume I can use "Next" to select the buttons but I don't want to try until I have confirmation that "Next" is the best option. If I get rid of the toolbar on most fields, I'll keep it for the DiveNumber field since it's a keypad without a built in return/done key.

Reply:
Having the method in the view controller is fine. In the future if there is tons of logic that could be pulled out of the view controller and moved to the custom view/control, that’s cool too. Generally you just want your view controllers to get the minimum info they need.

The “Next” button was just a little suggestion. Not necessary at all. Just go with the “Done” button where necessary for now.

Reply:
I liked the next idea until I started on in and realized I didn't know what to do about the buttons, toggles, and slider...

I removed the toolbar from fields that end editing with the return button for now.